unit-tests/live/safety/test-safety-metadata.py

```diff
@@ -0,0 +1,42 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#test:device D585S
+
+import pyrealsense2 as rs
+from rspy import test
+
+safety_metadata_values = [rs.frame_metadata_value.frame_counter,
```

Comment:
That's a good basic test.
I suggest adding a test that verify the data
Here we can see on a sequential frame the counter and timestamp is increasing as we expect it will.

Reply:
ok